portfolio/src/main/webapp/script.js

```diff
@@ -57,7 +56,109 @@ function createMapFunction() {
   window.initMap = function() {
     const map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 37.422403, lng: -122.088073}, zoom: 15});
+      {center: {lat: 37.422, lng: -122.084},
+      zoom: 16,
+      styles: [
+        {elementType: 'geometry', stylers: [{color: '#242f3e'}]},
```

Comment:
For readability, is it possible to use a datastructure for these colors. How I imagine it would be like is var colors = {"colorName1" : "#colorvalue1","colorName2" : "#colorvalue2"} and you can just refer to a color like colors["colorName1"].

It makes it more obvious to someone reading this code what the color is, and if you instead decide to use specific naming for keys, it makes it a little easier to change your color theme since you just have to update that data structure instead of finding all times you use a color value.

Reply:
Fixed in following commit.